draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 
 ### Unencrypted forwarder
 
-If the resolver announced by the network is a classic unencrypted forwarder, it is frequently the case that such forwarders are difficult to upgrade
-to support encrypted operation. In such cases it is useful for the resolver provider to be able to declare which encrypted resolvers they provide,
-and for the client to be able to discover them. If the client wishes to, it can then use one of those resolvers and bypass the local forwarder.
+If the resolver announced by the network is a classic unencrypted forwarder, it is frequently the case that such
+forwarders are difficult to upgrade to support encrypted operation. In such cases it is useful for the resolver
+provider to be able to declare which encrypted resolvers they provide, and for the client to be able to discover
+them. If the client wishes to, it can then use one of those resolvers and bypass the local forwarder.
 
 | Requirement | Description |
 | R3.1 | Example requirement |
 
 ### Encrypted forwarder
 
-If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver supports it. Discovering this
-may require communication with the local resolver, or an upstream resolver, over an unencrypted transport. Once discovered, the local encrypted forwarder
-may be selected by the client, gaining the benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local
-topology.
+If a subset of local resolvers supports encrypted DNS, the client may not initially be aware that its local resolver
+supports it. Discovering this may require communication with the local resolver, or an upstream resolver, over an
+unencrypted transport. Once discovered, the local encrypted forwarder may be selected by the client, gaining the
+benefits of encryption while retaining the benefits of a local caching forwarder with knowledge of the local topology.
 
 | Requirement | Description |
 | R4.1 | Example requirement |
 
-## Discover resolvers associated with a manually-configured resolver
+## Client-selected resolvers {#client-selected}
+
+Client devices often allow a user or administrator to select a specific DNS resolver to use
+on certain networks, or on all networks. Historically, this selection was specified only with an
+IP address.
+
+Discovering which equivalent encrypted resolvers are offered by the same entity allows the client
+to "upgrade" connections to use encrypted DNS.
+This can provide several benefits:
+
+- Prevent devices along the network path to the selected resolver from observing client DNS messages
+- Verify that answers come from the selected DNS resolver
+- Authenticate that the DNS resolver is the one selected by the client
+
+In doing so it is critical that the new resolver provides the same semantics as the original
+unencrypted one. Switching to one that provides different answers would break the expectation
+of the user who previously selected that resolver.
```

Comment:
I don't get this line (are you referring to a resolver performing DNS filtering) ?

Reply:
Yes. If the user configured 1.1.1.1, switching to 1.1.1.3 will break their expectations.

Reply:
Got it, Thanks.